media/css/m24/components/navigation-refresh.scss

```diff
@@ -158,10 +158,18 @@ $margin-top: 54px; // top margin offset for mobile navigation menu
     width: 100px;
 }
 
+
+
 // Mobile navigation menu with JS enabled.
 .js .m24-c-navigation-items {
     display: none;
-
+    
+    a {
+     &:hover {
+        color: $color-black !important;
+     }   
+    }
```

Comment:
Just quickly looking at the impact: _(not diving into formatting and linting as this would fail the checks, starting with important not being allowed generally etc.…)_

1. It's not always black, current item uses dark green so there are perhaps more combinations to check for.
2. I might have missed the idea why this is only applied in the js enhanced block, and not universally?
3. There are existing specificity overrides, so this might be worth exploring if such nesting helps with the order:

https://github.com/mozilla/bedrock/blob/b57052fc47f08fa3518aa82f9e33ab98bdb16634/media/css/m24/components/navigation-refresh.scss#L368-L373